AEPMessaging/Sources/schemas/JsonContentSchemaData.swift

```diff
@@ -17,7 +17,10 @@ import Foundation
 @objc(AEPJsonContentSchemaData)
 @objcMembers
 public class JsonContentSchemaData: NSObject, Codable {
+    /// Represents the content of the JsonContentSchemaData object.  Its value's type is determined by `format`.
     public let content: Any
+
+    /// Determines the value type of `content`.
```

Comment:
nit: `For now this value is always "application/json"`

Reply:
same here, i'm thinking i'll use `.applicationJson` - the value for `ContentType`